1_space_odyssey/index.html

```diff
@@ -10,7 +10,7 @@
 </head>
 <body class="fade-in">
     
-    <script src="./js/mannuel-9000/app.js"></script>
+    <script src="./js/mannuel-9000/farida-9000/app.js"></script>
```

Comment:
please put your named folder in the root of the JS folder at the moment it is in my named folder.